flink-yarn/src/main/java/org/apache/flink/yarn/YarnApplicationMasterRunner.java

```diff
@@ -72,26 +68,36 @@
  * This class is the executable entry point for the YARN application master.
  * It starts actor system and the actors for {@link org.apache.flink.runtime.jobmanager.JobManager}
  * and {@link YarnFlinkResourceManager}.
- * 
+ * <p>
```

Comment:
Of the 96 additions made a grand total of 6 are actually related to this PR. Please limit your auto-formatting. I'm not a fan of checking 15x the number of lines necessary.